pkgs/top-level/all-packages.nix

```diff
@@ -2164,7 +2164,6 @@ with pkgs;
 
   asymptote = callPackage ../tools/graphics/asymptote {
     texLive = texlive.combine { inherit (texlive) scheme-small epsf cm-super texinfo media9 ocgx2; };
-    gsl = gsl_1;
```

Comment:
Why was this here and why isn't it required anymore? Would be good to have that info in the commit message.